docs/examples.md

```diff
@@ -11,7 +11,7 @@
   "hookInstance": "d1577c69-dfbe-44ad-ba6d-3e05e953b2ea",
   "fhirServer": "http://fhir.example.com",
   "hook": "medication-prescribe",
-  "user": "Practitioner/example",
+  "user": "PractitionerRole/example",
```

Comment:
Throughout, remove "user" as a base-level field in the request body.  It only has a presence in individual hooks.

Reply:
Fixed, thanks, Dennis.